data/ubuntu-distro-info.json

```diff
@@ -265,19 +265,5 @@
     "eol": "2027-04-21",
     "eol_server": "2027-04-21",
     "eol_esm": "2032-04-21"
-  },
-  "v22.10": {
```

Comment:
@JamieMagee removed?

Reply:
Could be a mistake in upstream data. Some of the changes in Debian are also suspect.

Give it a day and manually re-run?

Reply:
I think we can close and wait for next week?